posts/maxistahl/01.open-ai-john-smith.md

```diff
@@ -47,6 +47,10 @@ I have learned from this exercise that even while I was interacting with John (A
 An AI does not have the ability to explore the world (yet), so many of the surprising answers it (he) gave me have a brief explanation of their nature: "I'm an AI, I have no opinion about...".
 What will happen in the world when an AI can not only be trained, but also learn from its context to perceive the value of what it learns?.
 
+We don't need to wait long to see this happen, developers all over the world are working on this, and we can start to see the results in integrations like OpenAI + Langchain + Python to mention at least one of the chosen ones (at the time of writing this post). This is actually one of our favorite stacks at CloudX Lab (we're working on several integration projects), discovering the potential of AIs by scanning the world for future products.
+
+We believe that these kinds of crazy ideas shape the shape of the future, the present with these tools is a huge step forward in the development of this technology.
```

Comment:
En vez de repetir shape pondría "these kinds of crazy ideas will shape the future". Luego en vez de coma, pondría punto y seguido para darles más enfasis a las oraciones.

Reply:
jaja un dedaso... fixeandooo

Reply:
arreglado (no sonaba tan mal en mi cabeza el shape the shape jaja)